src/helics/core/BrokerBase.cpp

```diff
@@ -28,7 +28,7 @@ SPDX-License-Identifier: BSD-3-Clause
 
 #    include <asio/steady_timer.hpp>
 #else
-#    ifdef _WIN32
+#    if defined(_WIN32) || defined(WIN32) || defined(__MINGW32__) || defined(CYGWIN)
```

Comment:
not sure about this one @nightlark does this look right to you as far as what conditions windows.h should be included?  Of course this is only if ASIO is disabled

Reply:
I think the conditions for this should be _WIN32 or WIN32 -- it seems like MSYS failure is because for some reason WIN32 is no longer being defined while _WIN32 is,  maybe something changed in the latest version.

```suggestion
#    if defined(_WIN32) || defined(WIN32)
```